lib/docker.ts

```diff
@@ -186,9 +193,19 @@ class Docker {
         }
       }
       if (!exclude) {
-        for (const g of globs) {
-          if (minimatch(filepath, g)) {
-            res.push(filepath);
+        if (globs.manifestGlobs) {
+          for (const g of globs.manifestGlobs) {
```

Comment:
is there a way that a file can match more than one glob (either manifest or binary)? I think we should stop the for loops if we find one matching glob, since it is super super expensive in CPU

Reply:
yep